src/Services/SmartProxy.js

```diff
@@ -25,11 +25,11 @@ export const SmartProxyApi = createApi({
     }),
     getClusters: builder.query({
       query: () => `v2/clusters`,
-    }),
-    getDisplayName: builder.query({
-      query: ({ id, includeDisabled }) =>
-        `v2/cluster/${id}/reports?get_disabled=${includeDisabled}`,
-      transformResponse: (response) => response?.report?.meta?.cluster_name,
+      transformResponse: (response) => {
+        return {
+          data: response.data.filter((element) => element.cluster_id !== ''),
```

Comment:
Nit-pick: can be rewritten to
```suggestion
          data: response.data.filter((element) => element.cluster_id),
```